Dynatrace/dynatrace-operator
# Make cert secret owned by the webhook deployment

Incase of google marketplace, its possible to change the `.Release.Name` of the helm chart, which should not be possible otherwise.

This makes it fail for `mpdev verify`.

Now instead we use the webhook deploment's name, which is static.
Furthermore the `webhook_cert_controller` reacts to the webhook deploment changes.
So in case the webhook deployment gets removed and added back, then the cert will be regenerated.

## How can this be tested?
Deploy, remove webhook deployment, check secrets, readd webhook deployment, check secrets


## Checklist
